protected/models/SurveyLanguageSetting.php

```diff
@@ -92,6 +92,8 @@ public function relations()
     public function rules()
     {
         return [
+            ['surveyls_survey_id','required', 'on' => 'insert'],
+            ['surveyls_language','required', 'on' => 'insert'],
```

Comment:
You can use CRangeValidator instead:
['surveyls_language', 'in', 'range'=> array_keys($this->survey->allLanguages)]